src/main/java/propertyFiles/geospatial.properties

```diff
@@ -283,3 +376,47 @@ controlledvocabulary.country.yemen=Yemen
 controlledvocabulary.country.zambia=Zambia
 controlledvocabulary.country.zimbabwe=Zimbabwe
 controlledvocabulary.country.aland_islands=Åland Islands
+controlledvocabulary.geometricObjectTypeCode.curve=curve
+controlledvocabulary.geometricObjectTypeCode.composite=composite
+controlledvocabulary.geometricObjectTypeCode.complex=complex
+controlledvocabulary.geometricObjectTypeCode.point=point
+controlledvocabulary.geometricObjectTypeCode.solid=solid
+controlledvocabulary.geometricObjectTypeCode.surface=surface
+controlledvocabulary.cellGeometryCode.point=point
+controlledvocabulary.cellGeometryCode.area=area
+controlledvocabulary.cellGeometryCode.voxel=voxel
+controlledvocabulary.dimensionNameTypeCode.row=row
+controlledvocabulary.dimensionNameTypeCode.column=column
+controlledvocabulary.dimensionNameTypeCode.vertical=vertical
+controlledvocabulary.dimensionNameTypeCode.track=track
+controlledvocabulary.dimensionNameTypeCode.crosstrack=crossTrack
+controlledvocabulary.dimensionNameTypeCode.line=line
+controlledvocabulary.dimensionNameTypeCode.sample=sample
+controlledvocabulary.dimensionNameTypeCode.time=time
+controlledvocabulary.spatialRepresentationType.stereomodel=stereoModel
+controlledvocabulary.spatialRepresentationType.video=video
+controlledvocabulary.spatialRepresentationType.tin=tin
+controlledvocabulary.spatialRepresentationType.texttable=textTable
+controlledvocabulary.spatialRepresentationType.grid=grid
+controlledvocabulary.spatialRepresentationType.vector=vector
+controlledvocabulary.dataQualityScope.dataset=dataset
+controlledvocabulary.dataQualityScope.service=service
+controlledvocabulary.geoResourceType.dataset=dataset
+controlledvocabulary.geoResourceType.service=service
+controlledvocabulary.geoResourceType.series=series
+controlledvocabulary.geoReferenceDateType.revision=revision
+controlledvocabulary.geoReferenceDateType.expiry=expiry
+controlledvocabulary.geoReferenceDateType.lastupdate=lastUpdate
+controlledvocabulary.geoReferenceDateType.lastrevision=lastRevision
+controlledvocabulary.geoReferenceDateType.nextupdate=nextUpdate
+controlledvocabulary.geoReferenceDateType.unavailable=unavailable
+controlledvocabulary.geoReferenceDateType.inforce=inForce
+controlledvocabulary.geoReferenceDateType.adopted=adopted
+controlledvocabulary.geoReferenceDateType.deprecated=deprecated
+controlledvocabulary.geoReferenceDateType.superseded=superseded
+controlledvocabulary.geoReferenceDateType.publication=publication
+controlledvocabulary.spatialResolutionType.equivalentscale=equivalentScale
+controlledvocabulary.spatialResolutionType.distance=distance
+controlledvocabulary.spatialResolutionType.vertical=vertical
+controlledvocabulary.spatialResolutionType.angulardistance=angularDistance
+controlledvocabulary.spatialResolutionType.levelofdetail=levelOfDetail
```

Comment:
Here at the bottom let's track failing tests. This is what I'm seeing right now:

edu.harvard.iq.dataverse.api.DatasetFieldsIT.testListAllFacetableDatasetFields	
edu.harvard.iq.dataverse.api.DatasetTypesIT.testUpdateDatasetTypeLinksWithMetadataBlocks
edu.harvard.iq.dataverse.api.DataversesIT.testListMetadataBlocks
edu.harvard.iq.dataverse.api.MetadataBlocksIT.testListMetadataBlocks

From a quick look they all have to do with the number of fields expected, like this:

```
JSON path data.size() doesn't match.
Expected: <64>
  Actual: <71>
```